api-admin/src/main/java/bio/terra/pearl/api/admin/controller/GlobalExceptionHandler.java

```diff
@@ -80,6 +85,10 @@ protected static ResponseEntity<ErrorReport> buildErrorReport(
         ex);
 
     return new ResponseEntity<>(
-        new ErrorReport().message(ex.getMessage()).statusCode(statusCode.value()), statusCode);
+        new ErrorReport()
```

Comment:
Are you comfortable having the admin and participant versions of GlobalExceptionHandler diverge?

Reply:
Yes, I would expect them to.  Also, since changes to Participant UI require about 10x more review and quality checks than admin UX, I like keeping controllers that might impact participant UI as separate as possible from admin controllers so we can iterate quickly on admin UX without thinking "will this trigger IRB scrutiny by impacting participant UI?"

Reply:
And to the concrete question of "should this particular  change be in the participant exception handler", I'd lean towards "not until we need it".  Since the participant UI is much more public, we should error on the side of not sending details to the UX